https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=828989

From: Reiner Herrmann <reiner@reiner-h.de>
Subject: ntp: please make the build reproducible
Date: Wed, 29 Jun 2016 16:43:31 +0200

While working on the "reproducible builds" effort [1], we have noticed
that ntp could not be built reproducibly.
It embeds the build time into the version string.

The attached patch removes this, so that it can be built reproducibly.

Index: ntp-4.2.8p10/scripts/build/mkver.in
===================================================================
--- ntp-4.2.8p10.orig/scripts/build/mkver.in
+++ ntp-4.2.8p10/scripts/build/mkver.in
@@ -15,8 +15,6 @@ case "@VER_SUFFIX@" in
  *) ConfStr="${ConfStr}-@VER_SUFFIX@" ;;
 esac
 
-ConfStr="$ConfStr `LC_TIME=C TZ=UTC date`"
-
 if [ ! -f .version ]; then
   echo 0 > .version
 fi
